docs/install.md

```diff
@@ -48,29 +27,31 @@ cd djangopackages
 
 In order to run the project, you'll need to run the following command:
 
-```bash
-docker-compose build --force-rm
-```
-
-### Add A GitHub API Token
+All of the environment variables and settings that are needed to run the project are stored in  `.env.local.example` file.
```

Comment:
@jefftriplett I would move this sentence to before "In order to run the project..." on line 28. That way, your statement, "you'll need to run the following command..." is immediately followed by the command, so the flow of instructions keeps going.

Reply:
danke